python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
```

Comment:
I would recommend defining the vowels outside of the `num_vowels()` and `num_consonants()` functions since you are defining this twice (DRY principle). Also, I'm curious, is "y" considered a vowel?